docs/robot.md

```diff
@@ -0,0 +1,136 @@
+# Clusters with Robot Servers
+
+## Quickstart
+
+Prerequisites:
+
+- Running Cluster
+- Kubectl
+- Helm
+
+0. Make sure that you start all Kubelets in the cluster with `--cloud-provider=external`
+
+1. Export credentials and create secret:
+
+   ```bash
+    export HCLOUD_TOKEN=your-hcloud-token
+    export ROBOT_USER=your-robot-user-name
+    export ROBOT_PASSWORD=your-robot-password
```

Comment:
Smart move, the space before the export commands!

(Will not be stored in the shell history)